Robitaille/template/simple_functions.py

```diff
@@ -10,3 +10,7 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+
+def IsPrime(value, option=None):
+    return value%2
```

Comment:
Please use `format` instead of `%`.  That's our style suggestion.  Obviously this is a string not a number.  I know that, and so do you, right?